server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -20,23 +20,37 @@ alfred:
   sessionStickinessDurationMs: 3600000
   enableConnectionCountLogging: false
   throttling:
-    restCallsPerTenant:
-        maxPerMs: 1000000
-        maxBurst: 1000000
-        minCooldownIntervalInMs: 1000000
-        minThrottleIntervalInMs: 1000000
-        maxInMemoryCacheSize: 1000
-        maxInMemoryCacheAgeInMs: 60000
-        enableEnhancedTelemetry: false
-    restCallsPerCluster:
-        createDoc: 
+    tenantGroup1:
+        createDoc:
+            maxPerMs: 1000000
+            maxBurst: 1000000
+            minCooldownIntervalInMs: 1000000
+            minThrottleIntervalInMs: 1000000
+            maxInMemoryCacheSize: 1000
+            maxInMemoryCacheAgeInMs: 60000
+            enableEnhancedTelemetry: false
+        getDeltas: disabled
+        getSession: disabled
+        socketConnections: disabled
+        submitOps: disabled
+    generalTenant:
+        generalRestCall: disabled
+        createDoc: disabled
+        getDeltas: disabled
+        getSession: disabled
+        socketConnections: disabled
+        submitOps: disabled
+    generalCluster:
```

Comment:
Let's call this perCluster. We are not splitting this to perTenant, so generalCluster does not make sense here

Reply:
Sounds good! Done. :)